index.html

```diff
@@ -0,0 +1,116 @@
+          <li><a href="#features" class="nav-link">Features</a></li>
+          <li><a href="#howItWorks" class="nav-link">How it works</a></li>
+          <li><a href="#pricing" class="nav-link">Pricing</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="main-section">
+      <div id="features" class="heading-input">
+        <div>
+          <h2>Handcrafted,Home-made masterpiece</h2>
+        </div>
+        <form action="https://www.freecodecamp.com/email-submit" id="form">
+          <input name="email"
+            type="email"
+            placeholder="Enter your Email address"
+            required
+            id="email"
+          />
+          <input id="submit" class="submit" type="submit" value="GET STARTED" />
+        </form>
+      </div>
+
+      <div id="howItWorks">
+        <div>
+          <h1>Premium Materials</h1>
+          <p>
+            Our trombones use the shiniest brass which is sourced locally. This
+            will increase the longevity of your purchase
+          </p>
+        </div>
+        <div>
+          <h1>Fast Shipping</h1>
+          <p>
+            We make sure you recieve your trombone as soon as we have finished
+            making it. We also provide free returns if you are not satisfied.
+          </p>
+        </div>
+        <div>
+          <h1>Quality Assurance</h1>
+          <p>
+            For every purchase you make, we will ensure there are no damages or
+            faults and we will check and test the pitch of your instrument.
+          </p>
+        </div>
+      </div>
+
+     
+        <iframe id="video"
+          width="560"
+          height="315"
+          src="https://www.youtube.com/embed/y8Yv4pnO7qc"
+          title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        ></iframe>
+
+      <div id="pricing" class="price">
+        <div>
+          <h1>TENOR TROMBONE</h1>
+          <h2>$600</h2>
+          <p>Lorem ipsum. <br>
+            Lorem ipsum. <br>
+            Lorem ipsum dolor. <br>
+            Lorem ipsum.</p>
+
+            <button type="button" class="submit">Select</button>
+        </div>
+        <div>
+          <h1>BASS TROMBONE</h1>
+          <h2>$900</h2>
+          <p>Lorem ipsum. <br>
+            Lorem ipsum. <br>
+            Lorem ipsum dolor. <br>
+            Lorem ipsum.</p>
+          <button type="button" class="submit">Select</button>
+        </div>
+        <div>
+          <h1>VALVE TROMBONE</h1>
+          <h2>$1200</h2>
+          <p>Lorem ipsum. <br>
+            Lorem ipsum. <br>
+            Lorem ipsum dolor. <br>
+            Lorem ipsum.</p>
+
+            <button type="button" class="submit">Select</button>
+        </div>
+      </div>
+    </section>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
```

Comment:
It is not recommended to add blank lines without a reason. Adding blank lines without a reason violates HTML best practices.

Reply:
- [ ] Excellent! :hugs: :clap: You indirectly told him what was wrong, and you brilliantly informed him about WHY it was wrong, but then, I believe you could make him understand you even better by leading him to make changes by actionable pleas :wink: A perfect example will be, `Kindly remove the blank lines`.
- With that, he will know the urgency of the need for action :+1: 
- [ ] It will be a lot more polite if you polarize the tone of your expression from commanding to appealing: Starting a communication with `It is not...` or ending it with `Adding blank lines without a reason violates HTML best practices.` might appear compelling, bossy or make the receiver feel inferior, and this not the impression you intend to pass across. Kindly rephrase to make it more conversational, friendly, polite, and encouraging :smile: